test/libsolidity/InlineAssembly.cpp

```diff
@@ -291,27 +320,27 @@ BOOST_AUTO_TEST_CASE(oversize_string_literals)
 
 BOOST_AUTO_TEST_CASE(assignment_after_tag)
 {
-	BOOST_CHECK(successParse("{ let x := 1 { tag: =: x } }"));
+	BOOST_CHECK(successParse("{ let x := 1 { 7 tag: =: x } }"));
```

Comment:
To clarify: `{ 2 { let x := 1 } pop }` and `{ let y := 2 { let x := 1} }` should produce identical bytecode.